### ISSUE TITLE
Server Class not updating 

### ISSUE THREAD

**Issue by pjsulin** [intersystems-community/vscode-objectscript#548]
I am running tune table on a class from the portal and the class in the ObjectScript Explorer will not update. I have tried refresh in the ObjectScript Explorer, closing VSCode, Refreshing the connection and toggling the connection, but do not see the tune table metadata. I am testing in parallel with Eclipse and there it is working fine.

I do notice that on Windows that hovering over the file in Object Explorer shows the local path to the same file. And now realize that if I make a change and try to save it tells me the version on server is different. Maybe this is just different behavior than I am used to, but in the ObjectScript Explorer I expect to see the contents of file on the Server. Right now File Explorer and Server Explorer seem to be showing the the same file.

How do I see the files on the server without updating my local files? I would expect that the File Explorere shows me local conttents and ObjectScript Explorer shows me what is on the server.

**Comment by isc-solon** [comment]
You need to right-click the file in ObjectScript Explorer and choose Export to get the updated server copy to your local machine.
I think it works like this:
- If you have a local copy of the file, File Explorer and ObjectScript Explorer both show the local version.
- If you don't have a local copy of the file, ObjectScript Explorer (obviously) shows the server version. Export brings it to your local machine.

**Comment by pjsulin** [comment]
I have the local files, which is causing the problem, as you stated in your first bullet point - The same file is shown in both. Why would I want to see the same file? FIle Explorer should show local file, ObjectScript Explorer should show files on Server.

**Comment by isc-bsaviano** [comment]
Seems like @pjsulin is expecting to see the same behavior as #494

**Comment by isc-bsaviano** [comment]
Closing as duplicate of #494
